procedures/unit-testing-utils.ipf

```diff
@@ -58,7 +58,7 @@ threadsafe static Function IsInteger(var)
 	return trunc(var) == var && numtype(var) == 0
 End
 
-// @brief Convert a text wave to string list
+/// @brief Convert a text wave to string list
```

Comment:
I know that this is a small fix, but please put unrelated fixes into a separate commit and PR.